developer/styleguide.md

```diff
@@ -1,119 +1,216 @@
-# Style Guide
+# Style guide
 
-Our rules are similar to AirBnB or Meteor rules, [standard template of ESLint rules](https://www.npmjs.com/package/eslint-config-airbnb), although we have tweaked them a little with what is working best for us.
+As a community, Reaction follows guidelines for JavaScript style and syntax conventions, along with guidelines for naming variables, methods and filenames.
 
-A couple of notable Reaction specific style rules:
+## Syntax and style conventions
 
-- Double quoted strings
-- Well spaced function
-- 160 character line length
+Our rules are similar to [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript) and [Meteor Code Style](https://guide.meteor.com/code-style.html), [standard template of ESLint rules](https://www.npmjs.com/package/eslint-config-airbnb), with a few custom Reaction-specific rules:
+
+- Double-quoted strings
+- Well-spaced functions
+- Spaces around brackets
+- 120 character line-length
 - `import` order
   - React npm packages (`React`, `prop-types`, etc...)
   - other npm packages
-  - meteor core packages
-  - meteor (Atmosphere) packages
+  - Meteor core packages
+  - Meteor (Atmosphere) packages
   - local app files
 
-Review [Meteor Code Style](https://guide.meteor.com/code-style.html) for additional guidelines that are typical of Meteor projects.
+Reaction uses various linting libraries to automate style checking. Find the exact rules in the code:
+- [`.eslintrc`](https://github.com/reactioncommerce/reaction/blob/master/.eslintrc) - [ESLint](http://eslint.org) checks JavaScript style, including [ES2015](https://docs.meteor.com/packages/ecmascript.html#Supported-ES2015-Features), React and Babel.
+- [`.jsbeautifyrc`](https://github.com/reactioncommerce/reaction/blob/master/.jsbeautifyrc) - [JS Beautifier](jsbeautifier.org) automates code formatting
+- [`.editorconfig`](https://github.com/reactioncommerce/reaction/blob/master/.editorconfig) - [Editor Config](https://editorconfig.org/) standardizes file formatting
+
+To see the rules in action, run `eslint .` from the command line or use [ESLint code editor tools](https://eslint.org/docs/user-guide/integrations).
 
-## Editor Configuration
+💡 **ProTip!** If you're using [Atom](https://atom.io/), like the Core team, you can install all the necessary tools in one line:  `apm install editorconfig atom-beautify linter linter-eslint linter-markdown linter-jsonlint linter-docker`
 
-In the Reaction app root, we have Reaction specific configuration files that can be used with most editors with the appropriate tools installed.
+## Naming conventions
 
-- `.eslintrc` - [http://eslint.org/](https://eslint.org/)
-- `.jsbeautifyrc` - [http://jsbeautifier.org/](https://jsbeautifier.org/)
-- `.editorconfig` - [http://editorconfig.org/](https://editorconfig.org/)
+### Avoid camel-casing files and folders
```

Comment:
I think that using positive language here might be more clear.
E.g. instead of `Avoid camel-casing files and folders`,  something like `Use hyphens to separate words in files and directories` or `File and directory names should be hyphenated`